locale/circuitpython.pot

```diff
@@ -1965,20 +1965,20 @@ msgstr ""
 msgid "The length of rgb_pins must be 6, 12, 18, 24, or 30"
 msgstr ""
 
-#: shared-module/audiomixer/MixerVoice.c
-msgid "The sample's bits_per_sample does not match the mixer's"
+#: shared-module/audiodelays/Echo.c shared-module/audiomixer/MixerVoice.c
+msgid "The sample's bits_per_sample does not match"
```

Comment:
We can join all of these messages together by using the QSTR for each attribute.

```suggestion
msgid "The sample's %q does not match"
```